modules/gltf/gltf_state.h

```diff
@@ -108,6 +108,7 @@ class GLTFState : public Resource {
 		HANDLE_BINARY_DISCARD_TEXTURES = 0,
 		HANDLE_BINARY_EXTRACT_TEXTURES,
 		HANDLE_BINARY_EMBED_AS_BASISU,
+		HANDLE_BINARY_EMBED_AS_UNCOMPRESSED,
```

Comment:
Probably needs a comment that if the enum value changes, the 3 hardcore in core needs to be changed too.

Reply:
I'll add a comment. It's still a bit icky